Makefile

```
@@ -415,15 +415,23 @@ lint_taxonomies:
 check_openapi_v2:
 	docker run --rm \
 		-v ${PWD}:/local openapitools/openapi-generator-cli validate --recommend \
-		-i /local/docs/api/ref/api.yml
+		-i /local/docs/api/ref/api.yaml
 
 check_openapi_v3:
 	docker run --rm \
 		-v ${PWD}:/local openapitools/openapi-generator-cli validate --recommend \
-		-i /local/docs/api/ref/api-v3.yml
+		-i /local/docs/api/ref/api-v3.yaml
 
 check_openapi: check_openapi_v2 check_openapi_v3
 
+lint_openapi:
+	@if [ "$(OS)" = "Darwin" ]; then \
+		echo "🥫 Linting OpenAPI is not supported on macOS"; \
+	else \
+		echo "🥫 Linting taxonomies"; \
+		docker run --rm -v $$(pwd):/app stoplight/spectral lint -r /app/.spectral.yaml /app/docs/api/ref/api.yml /app/docs/api/ref/api-v3.yml; \
```

you forgot to rename extension:

```suggestion
		docker run --rm -v $$(pwd):/app stoplight/spectral lint -r /app/.spectral.yaml /app/docs/api/ref/api.yaml /app/docs/api/ref/api-v3.yaml; \
```

With this I don't have the make lint_openapi passing:
```
$ docker run --rm -v $(pwd):/app stoplight/spectral lint -r /app/.spectral.yaml /app/docs/api/ref/api.yaml /app/docs/api/ref/api-v3.yaml

/app/docs/api/ref/api.yaml
   1:1     error  invalid-ref            '#/components/schemas/Ingredients' does not exist  properties.ingredients.items.properties.ingredients.$ref
 531:13  warning  oas3-unused-component  Potentially unused component has been detected.    components.schemas.Product
 533:16  warning  oas3-unused-component  Potentially unused component has been detected.    components.schemas.Ingredient
 536:14  warning  oas3-unused-component  Potentially unused component has been detected.    components.schemas.Nutrient

✖ 4 problems (1 error, 3 warnings, 0 infos, 0 hints)
```